ledger/participant-integration-api/src/main/scala/platform/apiserver/error/ErrorInterceptor.scala

```diff
@@ -70,13 +72,23 @@ final class ErrorInterceptor extends ServerInterceptor {
           val newMetadata =
             Option(Status.trailersFromThrowable(selfServiceException)).getOrElse(new Metadata())
           val newStatus = Status.fromThrowable(selfServiceException)
-          super.close(newStatus, newMetadata)
+          LogOnUnhandledFailureInClose(superClose(newStatus, newMetadata))
         } else {
-          super.close(status, trailers)
+          LogOnUnhandledFailureInClose(superClose(status, trailers))
         }
       }
 
+      /** This method serves as an accessor to the super.close() which facilitates its access from the outside of this class.
+        * This is needed in order to allow the call to be captured in the closure passed to the [[LogOnUnhandledFailureInClose]]
+        * error handler.
+        *
+        * TODO As of Scala 2.13.8, not using this redirection results in a runtime IllegalAccessError.
```

Comment:
As of means something like "2.13.8 and later", since it can be fixed later, can it be that you wanted to write "As at" or simply "At" instead?